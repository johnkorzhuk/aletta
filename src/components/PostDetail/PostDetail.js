import React, { PropTypes } from 'react'
const { object } = PropTypes

import Post from '../Post/Post'

const PostDetail = ({
  post
}) => {
  return (
    <div>
      <Post {...post} />
    </div>
  )
}

PostDetail.propTypes = {
  post: object
}

export default PostDetail
