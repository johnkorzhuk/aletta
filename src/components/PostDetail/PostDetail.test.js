/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import PostDetail from './PostDetail'

test('PostDetail component should render as expected with title as props', () => {
  const component = shallow(<PostDetail title='Some title' />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
