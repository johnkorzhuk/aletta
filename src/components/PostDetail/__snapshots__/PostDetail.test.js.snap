// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostDetail component should render as expected with title as props 1`] = `
<div>
  <Post />
</div>
`;
